components/eam/src/physics/cam/micro_p3_interface.F90

```diff
@@ -373,13 +348,10 @@ subroutine get_prescribed_CCN_from_file(micro_p3_lookup_dir,month_int, ccn_value
       write(mon_str,*) month_int
 
       !assign base_file_name the name of the CCN file being used 
-      base_file_name = "prescribed_CCN_file_"
+      base_file_name = "spa_file_"
```

Comment:
Have we updated these filenames on the machines we are running on?

Reply:
Yes, I'm in the process of making these files available for all machines.